django/db/backends/base/schema.py

```diff
@@ -407,14 +407,12 @@ def add_field(self, model, field):
         # Drop the default if we need to
         # (Django usually does not use in-database defaults)
         if not self.skip_default(field) and self.effective_default(field) is not None:
+            sql, params = self._alter_column_default_sql(model, None, field, True)
```

Comment:
Use `drop=True` rather than an arg for better readability.
Use a different var for `sql`here, maybe `changes_sql`? It's a little confusing to have the `sql` name reused in the next line.